crates/primitives/src/block.rs

```diff
@@ -612,19 +612,19 @@ impl<T: Transaction> BlockBody<T> {
     /// Returns whether or not the block body contains any blob transactions.
     #[inline]
     pub fn has_blob_transactions(&self) -> bool {
-        self.transactions.iter().any(|tx| tx.is_eip4844())
+        self.transactions.iter().any(|tx| Typed2718::is_eip4844(tx))
```

Comment:
let's just remove those methods from the reth `Transaction` trait https://github.com/paradigmxyz/reth/blob/fcff53a6f8aa22cba143ed2202e93be7129f9aee/crates/primitives-traits/src/transaction/mod.rs#L35-L63

then we can invoke this normally as just `tx.is_eip4844()`

Reply:
thanks, fixed it in https://github.com/paradigmxyz/reth/pull/13268/commits/fab7998728860c23f4852b29501944834a975d7d